integration-cli/docker_cli_build_test.go

```diff
@@ -1986,8 +1986,7 @@ func TestBuildCancelationKillsSleep(t *testing.T) {
 	eventStart := make(chan struct{})
 	eventDie := make(chan struct{})
 
-	// Start one second ago, to avoid rounding problems
-	startEpoch := time.Now().Add(-1 * time.Second)
+	startEpoch := daemonTime(t).Unix()
```

Comment:
I can understand that using daemonTime() would reduce the race stuff, but I was wondering about the "...to avoid rounding problems" part of the comment.  Not sure if the reason they wrote that goes away once we get the time from the daemon.  @pwaller  ?